src/main/java/io/github/aquerr/steamwebapiclient/request/PlayerGetRecentlyPlayedGamesRequest.java

```diff
@@ -0,0 +1,46 @@
+package io.github.aquerr.steamwebapiclient.request;
+
+import io.github.aquerr.steamwebapiclient.annotation.SteamRequestQueryParam;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * The request representing <a href="https://partner.steamgames.com/doc/webapi/IPublishedFileService#QueryFiles">https://partner.steamgames.com/doc/webapi/IPublishedFileService#QueryFiles</a>
+ */
+@Builder
+@Getter
+@Setter
+@ToString
+public class PlayerGetRecentlyPlayedGamesRequest implements SteamWebApiRestrictedRequest {
```

Comment:
Hmmm... I think it would be better if we could get rid of "Get" part.
I would recommend to change it to "PlayerRecentlyPlayedGamesRequest"

Reply:
fixed